packages/common/src/audius-query/createApi.ts

```diff
@@ -127,8 +127,10 @@ export const createApi = <
       (endpointName, fetchArgs, updateRecipe) =>
       (dispatch: Dispatch, getState: () => any) => {
         const key = getKeyFromFetchArgs(fetchArgs)
-        const endpointState = getState().api[reducerPath][endpointName][key]
-        if (!endpointState) return
+        let endpointState = getState().api[reducerPath][endpointName][key]
+        if (!endpointState) {
+          endpointState = { status: Status.SUCCESS, normalizedData: {} }
+        }
```

Comment:
Change @amendelsohn and I added to init data in the cases like this one where we want to populate content of a different query but that query hasn't been called yet